app/src/main/java/ar/scacchipa/twittercloneapp/data/repository/LoggedUserRepository.kt

```diff
@@ -0,0 +1,68 @@
+package ar.scacchipa.twittercloneapp.data.repository
+
+import ar.scacchipa.twittercloneapp.data.datasource.ILocalSource
+import ar.scacchipa.twittercloneapp.data.datasource.ILoggedUserExternalSource
+import ar.scacchipa.twittercloneapp.data.model.UserData
+import ar.scacchipa.twittercloneapp.utils.Constants.Companion.LOGGED_USER_DATA_ID
+import ar.scacchipa.twittercloneapp.utils.Constants.Companion.LOGGED_USER_DATA_NAME
+import ar.scacchipa.twittercloneapp.utils.Constants.Companion.LOGGED_USER_DATA_PROFILE_IMAGE_URL
+import ar.scacchipa.twittercloneapp.utils.Constants.Companion.LOGGED_USER_DATA_USERNAME
+import ar.scacchipa.twittercloneapp.utils.Constants.Companion.LOGGED_USER_DATA_VERIFIED
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.coroutineScope
+import kotlinx.coroutines.launch
+
+class LoggedUserRepository(
+    private val loggedUserLocalSource: ILocalSource,
+    private val loggedUserExternalSource: ILoggedUserExternalSource,
+    private val credentialRepository: ICredentialRepository,
```

Comment:
I think this repository dependency should not be here.

Reply:
The credentialRepository dependency was removed. The accessToken is transferred by parameter.